packages/types/src/types/requests/datatrak-web-server/UserRequest.ts

```diff
@@ -10,8 +10,9 @@ export interface ResBody {
   id?: string;
   userName?: string;
   email?: string;
-  projectId?: string;
+  preferences?: any;
   project?: ProjectResponse | null;
+  projectId?: string;
```

Comment:
Yes the user route still returns the projectId field for convenience on the front end since it's used in a bunch of places for conditional rendering. I'm not sure if that's bad practise but it seemed like the orchestration server making things convenient for the front end is part of their role